data/World/Overworld.json

```diff
@@ -594,7 +647,37 @@
             "GF GS Top Floor": "
                 is_adult and at_night and 
                 (Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot) or
-                    can_use(Hover_Boots) or logic_gerudo_kitchen)"
+                    can_use(Hover_Boots) or logic_gerudo_kitchen)",
+            "GF Above Jail Crate": "can_use(Scarecrow) or can_use(Longshot)",
+            "Hideout Break Room Pot 1": "(can_use(Hover_Boots) or can_use(Scarecrow) or can_use(Longshot)) and (Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot))",
+            "Hideout Break Room Pot 2": "(can_use(Hover_Boots) or can_use(Scarecrow) or can_use(Longshot)) and (Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot))",
+            "Hideout Break Room Crate 1": "(can_use(Hover_Boots) or can_use(Scarecrow) or can_use(Longshot)) and (Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot))",
+            "Hideout Break Room Crate 2": "(can_use(Hover_Boots) or can_use(Scarecrow) or can_use(Longshot)) and (Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot))",
+            "Hideout Break Room Crate 3": "(can_use(Hover_Boots) or can_use(Scarecrow) or can_use(Longshot)) and (Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot))",
+            "Hideout Break Room Crate 4": "(can_use(Hover_Boots) or can_use(Scarecrow) or can_use(Longshot)) and (Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot))",
+            "Hideout 1 Torch Jail Pot 1": "True",
+            "Hideout 1 Torch Jail Pot 2": "True",
+            "Hideout 1 Torch Jail Pot 3": "True",
+            "Hideout 1 Torch Jail Crate": "True",
+            "Hideout Kitchen Pot 1": "Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot)",
+            "Hideout Kitchen Pot 2": "Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot)",
+            "Hideout Near Kitchen Crate 1": "Gerudo_Membership_Card or can_use(Bow) or can_use(Hookshot)",
+            "Hideout Near Kitchen Crate 2": "True",
```

Comment:
We have a trick "Thieves' Hideout 'Kitchen' with No Additional Items" that should apply to every difficult check in the kitchen, as both ages, with no extra items required. I tested and I was able to collect all items with nothing extra. As child I usually got caught immediately after lol, but it's doable. You just gotta bring the jukes.

This trick was never intended to be so difficult -- I originally intended it for it to just be for the glitchless jump outside. Since then, it's become a monster. I have plans soon to rework Gerudo fortress. At the least I want to separate the glitchless jump outside from the super stealth inside. (I think it might be a good idea to actually separate this place apart by room for ER compatability even if these rooms will never be ER'd? And it might help organize your new checks a little if Fortress were separated in this way.)

Even without any tricks enabled, you might want to consider allowing using the Boomerang on the pots. As long as the items stick right away and they often do, you can just stand by the upper entrance and bring the items to you.

For reaching the break room, there is a trick jump you could add that can be used to get over without Hovers or Hookshot. Stand below where the wall skulltula is facing the left wall near the edge, and do a ESS shuffle while targeting the wall, and then quickly turn and roll to jump while still facing the wall. Then you can grab ledge of the ground over there and run over to the break room entrance. Only works as adult. (You can also just jump from the jail but there's no way back in after getting Gerudo Card. Non-repeatable access is generally something to be avoided but since this interior isn't shuffled it's probably okay?)

The first two crates in the break room seem to be obtainable without needing an items to reach with guards. The one guard that walks around has to not be too close, but seems final overall. Of course you will need to pass the kitchen (Bow, Hookshot, Hovers, Card, or trick), but once you're here they should be obtainable for free.

So first guard in here is easy enough to dispatch with nothing, but the other one is impervious to swords for some reaon. If it was just the first guard that was guarding the back two crates, I might argue that it was safe for default to dispatch her with just the sword, since sometimes we've considered taking out a single guard to be safe. But unfortunately for the back two crates, the impervious guard can kind of get in the way a little bit. The right crate is generally okay as long as you don't walk fully to the back of the room. The left crate is doable if you bonk it from the very edge, and then roll to the item underneath her vision. You could probably fold this into the current "with No Additional Items" trick with minimal changes its name and tooltip?

That guard though, she guards the two pots on the table with her life. And yet it's totally possible to get them. You can roll under her vision behind the table, then clip up onto it, and just stand there like everything's fine. The pot farther from her side of the table you can actually just pick up and cart away. The pot closer to her however you unfortunatly have to rely on the item drop to not fall onto her lap after slashing the pot. You can't quite get close enough to pick it up without getting caught. All-in-all, a really strange room. I'm really not sure how far I want to be pushing this kitchen trick or how to handle all of these weird interactions. I think just do to the jank and some of the weirdness with RNG, I would leave these two pots out of any trick? But I'm not sure. (I'm not sure if how item drops move has been adjusted in this build. Certainly some of what I've said here should probably be tested on the real thing, rather than with GZ, just in case something has changed in a significant way.)

Reply:
Alright after having gone through the whole PR I'm ready to revisit this comment. Here are my recommendations for the changes you should make at this time:

1) It should be in logic for child to Boomerang the pots in the Kitchen.
2) The first two crates in the break room need not check for a means of stunning the guard. You should name them differently from the other two crates since the logic for them will be different.
3) All of the checks in the Kitchen should be in logic under the trick 'logic_gerudo_kitchen' with no additional items required. The trick was never intended to be this difficult and I plan to rework how the fortress tricks work in the future, but for now I would just throw these pots and crates in with it for consistency. Since the break room is not the kitchen, don't worry about adding tricks to the checks in there.

And, notes for future me, some possible tricks:
1) Trick jumps outside to climb to the top fortress.
2) Trick jump outside to reach the break room.
3) New trick for Break room guard dodges, or rework Kitchen dodges trick to include Break room. (Perhaps the final pot though is too annoying to be covered under any trick.)